www/include/configuration/configServers/DB-Func.php

```diff
@@ -650,6 +650,12 @@ function insertServer(array $data): int
     $poller = $result->fetch();
     $result->closeCursor();
 
+    try {
+        insertServerIntoPlatformTopology($retValue, (int) $poller['last_id']);
+    } catch (Exception $e) {
```

Comment:
Do you want to catch all types of exception, or only `\PDOException` ?

Reply:
PDOException and Exception that could be thrown while resolving the parent relation:
```
        if (!empty($parent['id'])) {
            $parentId = (int) $parent['id'];
        } else {
            throw new \Exception(
                'Missing parent platform topology. Please register the parent first using the endpoint
                or the available script. For more details check the documentation'
            );
        }
```